mmv1/products/memorystore/InstanceDesiredUserCreatedConnections.yaml

```diff
@@ -0,0 +1,134 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'InstanceDesiredUserCreatedConnections'
+api_resource_type_kind: Cluster
```

Comment:
Not sure of the significance of this field but this is binding resource for Memorystore instance and not cluster